core-requirements.txt

```diff
@@ -10,3 +10,4 @@ psutil>=5.6.3
 requirements-parser>=0.2.0
 shap>=0.35.0
 texttable>=1.6.2
+woodwork>=0.0.3
```

Comment:
@gsheni When will woodwork be added to conda forge?

Reply:
Working on this currently, and hopefully will have more of an update end of this week.

Reply:
Thank you so much!

Reply:
🎊